pkg/blobstore/configuration/new_blob_access.go

```diff
@@ -143,6 +144,35 @@ func (nc *simpleNestedBlobAccessCreator) newNestedBlobAccessBare(configuration *
 			BlobAccess:      mirrored.NewMirroredBlobAccess(backendA.BlobAccess, backendB.BlobAccess, replicatorAToB, replicatorBToA),
 			DigestKeyFormat: backendA.DigestKeyFormat.Combine(backendB.DigestKeyFormat),
 		}, "mirrored", nil
+    case *pb.BlobAccessConfiguration_Quorum:
+        backends := make([]blobstore.BlobAccess, 0, len(backend.Quorum.Backends))
+		var combinedDigestKeyFormat *digest.KeyFormat
+
+        for _, b := range backend.Quorum.Backends {
+            backend, err := nc.NewNestedBlobAccess(b, creator)
+            if err != nil {
+                return BlobAccessInfo{}, "", err
+            }
+            backends = append(backends, backend.BlobAccess)
+            if combinedDigestKeyFormat == nil {
+                combinedDigestKeyFormat = &backend.DigestKeyFormat
+            } else {
+                newDigestKeyFormat := combinedDigestKeyFormat.Combine(backend.DigestKeyFormat)
+                combinedDigestKeyFormat = &newDigestKeyFormat
+            }
+        }
+		if len(backends) == 0 {
+			return BlobAccessInfo{}, "", status.Errorf(codes.InvalidArgument, "Cannot create quorum blob access without any backends")
+		}
+        readQuorum := int(backend.Quorum.ReadQuorum)
+        writeQuorum := int(backend.Quorum.WriteQuorum)
+        if readQuorum + writeQuorum <= len(backends) {
+			return BlobAccessInfo{}, "", status.Errorf(codes.InvalidArgument, "Quorum blob access requires read_quorum + write_quorum > number of backends")
+        }
```

Comment:
Yeah, I felt it was far more likely for someone to mistakenly put a small negative number in here rather than a super large positive number - so I went with a uint in the proto.  I agree I don't like casting int types and I'm happy to change this to a int throughout. (Or whatever other solution you would prefer)